src/tests/intg/test_infopipe.py

```diff
@@ -160,8 +160,8 @@ def cleanup_ldap_entries(ldap_conn, ent_list=None):
     """Remove LDAP entries added by create_ldap_entries"""
     if ent_list is None:
         for ou in ("Users", "Groups", "Netgroups", "Services", "Policies"):
-            for entry in ldap_conn.search_s("ou=" + ou + "," +
-                                            ldap_conn.ds_inst.base_dn,
+            for entry in ldap_conn.search_s("ou=%s,%s" % (ou,
+                                            ldap_conn.ds_inst.base_dn),
```

Comment:
Can you use the new `f-Strings` feature to concatenate those strings? I think it will be easier to read than your proposal.

This also applies to the next changes.

Reply:
done